app/controllers/users_controller.rb

```diff
@@ -29,7 +29,12 @@ def assign_role
   end
 
   def show
-    @sales = current_user.events
+    @events = current_user.events
+    @my_orders = Order.where(customer_id: current_user.id)
+  end
+
+  def index
+    @users = User.all.select{|user| user.store_admin?}
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:store_admin?` as an argument to `select` instead of a block.